sql/src/test/java/ddlog/JooqProviderTestBase.java

```diff
@@ -132,6 +113,7 @@ protected void starting(Description description) {
      */
     @Test
     public void testSqlOpsNoBindings() {
+        Assume.assumeTrue(this.getClass() != JooqProviderTestBase.class);
```

Comment:
How about moving this code to a one-line function that you call everywhere?
Can this be done with a Rule?